### ISSUE TITLE
Runners want to be bipedal on Curseforge 

### ISSUE THREAD

**Issue by sonofabaconator** [cybercat-mods/gigeresque#161]
I downloaded the mod on curseforge and I was doing my brutal experiments (as per usual) but when I infected an animal, the runnerburster would grow up to be a Drone instead of a Runner.  Not sure if this was already known and I apologize if it already was.

**Comment by AzureDoom** [comment]
Need versions and what mobs tested again.

**Comment by sonofabaconator** [comment]
Edit: Found out that if I spawn a runnerburster with an egg or summon it with commands, it grows up to be a Drone

okay... so I just went and tested pretty much every quadrupedal mob in the game and they all came out as runners. But I swear to you I infected a wolf one time and a drone came out.

P.S. I noticed a lot of pathfinding issues with the facehuggers, I had them in 5x5 glass boxes with the animal and it took it like 30 seconds of crawling on the walls and jumping around to finally facehug the animal.

P.P.S. I'm not sure what your roadmap is, but I think it would be a cool idea to have spitters come out of llamas
